editor/src/clj/editor/settings_core.clj

```diff
@@ -266,11 +307,33 @@
     (str value "c")
     value))
 
-(defmethod render-raw-setting-value :default [_ value]
-  (str value))
+(defmulti quoted-list-element-setting? (fn [meta-setting] (:type meta-setting)))
+
+(defmethod quoted-list-element-setting? :default [_] false)
+
+(defmethod quoted-list-element-setting? :string [_] true)
+
+(defmethod quoted-list-element-setting? :resource [_] true)
+
+(defmethod quoted-list-element-setting? :file [_] true)
+
+(defmethod quoted-list-element-setting? :directory [_] true)
+
+(defmethod quoted-list-element-setting? :url [_] true)
```

Comment:
I wonder if this should be a single function instead of a multi-method, or perhaps, even a map?